### PR TITLE
docs: add pop api documentation

### PR DESCRIPTION
**Pull request by brunopgalvao**
ditto

Please be hard on the feedback so we can make this perfect.

### CODE REVIEW

File: pop-api/README.md
Comment (Daanvdplas) [inline pop-api/README.md:14]: I have my reservations here. Not sure whether they are comparable, and the relevance of comparing it with the "runtime api of the outer node."

**Review by Daanvdplas — COMMENTED** [review]
Nicely done!

**Comment by evilrobot-01** [comment]
Sadly I think we should wait until we downgrade `pop-api` to ink! v4 before we can merge this.

**Comment by brunopgalvao** [comment]
> Sadly I think we should wait until we downgrade `pop-api` to ink! v4 before we can merge this.

Bumping the PR. Now that we will not be downgrading to `inkv4` I believe we can merge this.

File: pop-api/README.md
Comment (Daanvdplas) [reply pop-api/README.md:14]: Still think that this whole bit is unnecessary. It is complicated and not really something a smart contract developer needs to understand.

File: pop-api/README.md
Comment (Daanvdplas) [inline pop-api/README.md:46]: `pop_api::nfts` is not necessary if you look at the examples.

File: pop-api/README.md
Comment (Daanvdplas) [inline pop-api/README.md:49]: "nice" => "user friendly"

File: pop-api/README.md
Comment (Daanvdplas) [inline pop-api/README.md:37]: Remove section. Not very important to understand for the smart contract developer and not theoretically true since it is much more than just the types in primitives

File: pop-api/README.md
Comment (brunopgalvao) [reply pop-api/README.md]: Updated.

File: pop-api/README.md
Comment (brunopgalvao) [reply pop-api/README.md:37]: I disagree. This is part is essential. It is the bridge btwn ink! pop-api and runtime pop-api.

**Review by Daanvdplas — APPROVED** [review]

File: pop-api/README.md
Comment (Daanvdplas) [reply pop-api/README.md:14]: It is not special to pop api of pop network though. This is already possible since chain extensions were developed

File: pop-api/README.md
Comment (Daanvdplas) [reply pop-api/README.md:37]: These types can also be found in the pop api so you would have to add that then.

**Review by AlexD10S — APPROVED** [review]
Nice! 
I suggest you add this text (Maybe without the links to the code) in our Documentation page too: https://learn.onpop.io/pop/welcome/pop-api

**Review by al3mart — APPROVED** [review]

**Pull request merged**
